integration-tests/test/conftest.py

```diff
@@ -80,18 +82,24 @@ def temporary_bonds_file(random_generator: Random, validator_keys: List[PrivateK
         os.unlink(file)
 
 
-def make_wallets_file_lines(random_generator: Random, validator_keys: List[PrivateKey]) -> List[str]:
+def make_wallets_file_lines(wallet_keys_map: Dict[PrivateKey, int]) -> List[str]:
```

Comment:
Nitpick: How about `wallet_balance_from_private_key` instead of `wallet_keys_map`?  Same goes for all the other places in the code.